### PR TITLE
added blog post about socrates germany 2018

### PR DESCRIPTION
**Pull request by jaramir**

### CODE REVIEW

File: _posts/2018-08-28-socrates-germany-2018.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+[SoCraTes Germany 2018](https://www.socrates-conference.de) is over and it has been a great one. Many insights, new ideas, and opportunities to reflect. Here are my notes.
+
+# Saturday
+
+It started with learning how to make a coffee with an [Aeropress and KanBan](https://twitter.com/AnnNat/status/1033015354967121921). A delicious example of how feedback loops and reflection can make you go faster.
+
+Next I tried sketching and we discussed how sketching can be used for note taking but it can also aid communication, for example during a meeting or in a presentation. It creates visual memories that can be referred to later. The header picture is my own attempt at it.
+
+Non symbolic drawing was a surprisingly challenging session for me.
+We don't normally see with our eyes, we see with our brain. The first exercise was to draw the negative space of an object, eg. The "holes" around a wrapped up cable. It is an experience in seeing.
```
Comment (alvarogarcia7) [inline _posts/2018-08-28-socrates-germany-2018.md:22]: Are these different paragraphs, or just a big one?

File: _posts/2018-08-28-socrates-germany-2018.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Refuctoring was another "technical" session: start with a plain Fizz-Buzz implementation in JavaScript and turn it into the most [diabolically intricate piece of code](https://github.com/Hackerkegeln/refuctoring/blob/socratees2018/src/fizz-buzz-generator.js) we could. The insight for me happened when the code crossed my mental ability to model its behaviour, which happened upon inlining a function.
+Immediately after my brain just gave up, my mental model was gone, garbage collected. My mind was empty and it took me a good 10 minutes to understand that part of the code again. To recreate a mental model.
+Fascinating.
+
+There was more, we talked about the five languages of love, [played board games](https://twitter.com/coderbyheart/status/1033076918160162819), heard about the book Accelerate, discussed code reviews and continuous delivery, and collected a lot of laptop stickers.
+
+I had an amazing time and I'm so grateful to the organisers and everyone that come to SoCraTes Germany this year. [Franzi](https://twitter.com/Singsalad) in particular, for being a great travel buddy and facilitator.
+
+See you again in a year!
```
Comment (alvarogarcia7) [inline _posts/2018-08-28-socrates-germany-2018.md:47]: Would you like to mention the workshops on Sun?

**Review by alvarogarcia7 — APPROVED** [review]

File: _posts/2018-08-28-socrates-germany-2018.md (hunk 1 above)
Comment (jaramir) [reply _posts/2018-08-28-socrates-germany-2018.md:22]: two different paragraphs

File: _posts/2018-08-28-socrates-germany-2018.md (hunk 2 above)
Comment (jaramir) [reply _posts/2018-08-28-socrates-germany-2018.md:47]: I did not do any :)

Comment (alvarogarcia7) [reply _posts/2018-08-28-socrates-germany-2018.md:47]: I remember :)

**Review by rachelcarmena — APPROVED** [review]
Approved! Thanks for sharing the experience @jaramir !!

**Pull request merged**